base/document-download-api-deployment.yaml

```diff
@@ -3,14 +3,19 @@ kind: Deployment
 metadata:
   labels:
     app: document-download-api
-  name:  document-download-api
+  name: document-download-api
   namespace: notification-canada-ca
 spec:
   replicas: 1
   revisionHistoryLimit: 5
   selector:
     matchLabels:
       app: document-download-api
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
```

Comment:
Those are the defaults, but now it's explicit. https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#rolling-update-deployment 

Because we're running 1 replica, `maxSurge` will be 1 and `maxUnavailable` will be 0